CHANGELOG.md

```diff
@@ -150,6 +152,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - `GetParent` will now always return null for objects with no parents, instead of returning the self object for things that weren't `Attachables`. This makes things more consistent and reasonable throughout and will rarely, if ever, cause Lua problems.
 
+- BuyMenu tooltips now display item info as well as a description. This includes the maximum carriable mass and passengers for Craft, mass and passenger slots required for Actors, and mass for other Moveable Objects.
```

Comment:
Formatting tweaks needed. For anything that's an actual property or class, we write it out as such and surround it in ``s. I would also suggest re-ordering things a little bit too.
I.e.
- The `Actor` property `MaxMass` now no longer includes the `Mass` of the `Actor`, and has been renamed to `MaxInventoryMass` for clarity. In mods, this is most important for `ACraft`, which will now need their total `Mass` subtracted from the value.

- `BuyMenu` tooltips now display useful item info as well as their description. This includes `MaxInventoryMass` and `MaxPassengers` for `ACraft`, `Mass` and `PassengerSlots` required for `Actors`, and `Mass` for other `MoveableObjects`.